wagtail/contrib/search_promotions/migrations/0007_merge_20230915_0846.py

```diff
@@ -0,0 +1,14 @@
+# Generated by Django 4.2.5 on 2023-09-15 08:46
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('wagtailsearchpromotions', '0006_reset_query_sequence'),
+        ('wagtailsearchpromotions', '0006_searchpromotion_external_link_title_and_more'),
+    ]
+
+    operations = [
+    ]
```

Comment:
This migration does nothing, I think it can be removed once the previous migration is fixed up.